src/api/app/models/event/request.rb

```diff
@@ -71,10 +71,7 @@ def calculate_diff(a)
   end
 
   def source_from_remote?
-    payload['actions'].each do |action|
-      return true if Project.unscoped.is_remote_project?(action['sourceproject'], true)
-    end
-    false
+    payload['actions'].any? { |action| Project.unscoped.is_remote_project?(action['sourceproject'], true) }
```

Comment:
This would return `nil` in case there are `actions` but none of them is remote. We deliberately avoided this...

Reply:
No, it will always return true or false. Check out the [docu](https://ruby-doc.org/core-2.4.1/Enumerable.html):-)

Reply:
I also don't see a problem with it.